Tests/mocks/devicesettings.h

```diff
@@ -1208,3 +1208,319 @@ class FrontPanelConfig {
 
 }
 
+
+#define RT_OK 0
```

Comment:
is it part of devicesettings?

Reply:
RT_OK, RT_ERROR and so on are apart of the rtError class in the base code, which are included in the various files where the values are used(xCast, and the rtObject classes). I can move it elsewhere, if you'd prefer it to be somewhere else.